src/java.base/share/classes/java/lang/Math.java

```diff
@@ -1007,6 +1002,53 @@ public static long multiplyExact(long x, long y) {
         return r;
     }
 
+    /**
+     * Returns the quotient of the arguments, throwing an exception if the
+     * result overflows an {@code int}.  Such overflow can occur if and only
+     * if either {@code y} is zero, or both {@code x} is
```

Comment:
I think the divide by zero case should be discussed separately from overflow.

Reply:
Fixed.